content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+Untuk informasi mengenai cara membuat sebuah klaster dengan kubeadm setelah kamu melakukan proses instalasi ini, lihat laman [Menggunakan kubeadm untuk Membuat Sebuah Klaster](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+* Satu mesin atau lebih yang menjalankan:
+  - Ubuntu 16.04+
+  - Debian 9+
+  - CentOS 7
+  - Red Hat Enterprise Linux (RHEL) 7
+  - Fedora 25+
+  - HypriotOS v1.0.1+
+  - Container Linux (teruji pada versi 1800.6.0)
+* 2 GB RAM atau lebih per mesin (kurang dari nilai tersebut akan menyisakan sedikit ruang untuk
+   aplikasi-aplikasimu)
+* 2 CPU atau lebih
+* Koneksi internet pada seluruh mesin pada klaster (kamu dapat menggunakan internet
+  publik ataupun pribadi)
+* _Hostname_ yang unik, alamat MAC, dan product_uuid untuk setiap Node. Lihat [di sini](#memastikan-alamat-mac) untuk detail lebih lanjut.
+* Port tertentu pada mesin. Lihat [di sini](#check-required-ports) untuk detail lebih lanjut.
+* _Swap_ dinonaktifkan. Kamu **HARUS** menonaktifkan _swap_ agar kubelet dapat berfungsi dengan baik.
+
+
+
+<!-- steps -->
+
+## Memastikan alamat MAC dan product_uuid yang unik untuk setiap Node {#memastikan-alamat-mac}
+
+* Kamu bisa mendapatkan alamat MAC dari antarmuka jaringan menggunakan perintah `ip link` atau `ifconfig -a`
+* product_uuid didapatkan dengan perintah `sudo cat /sys/class/dmi/id/product_uuid`
+
+Sangat memungkinkan bagi perangkat keras untuk memiliki alamat yang unik, namun beberapa mesin virtual bisa memiliki
+nilai yang identik. Kubernetes menggunakan nilai-nilai tersebut untuk mengidentifikasi Node-Node secara unik pada klaster.
+Jika nilai-nilai tersebut tidak unik pada tiap Node, proses instalasi
+bisa saja [gagal](https://github.com/kubernetes/kubeadm/issues/31).
+
+## Memeriksa adaptor jaringan 
+
+Jika kamu memiliki lebih dari satu adaptor jaringan, dan komponen Kubernetes tidak dapat dijangkau melalui rute bawaan (_default route_),
+kami merekomendasikan kamu untuk menambahkan rute IP sehingga alamat-alamat klaster Kubernetes melewati adaptor yang tepat.
+
+## Membuat iptables melihat _bridged traffic_
+
+Agar iptables pada Node Linux dapat melihat _bridged traffic_ dengan benar, kamu harus memastikan `net.bridge.bridge-nf-call-iptables` bernilai 1 pada pengaturan `sysctl`, misalnya.
+
+```bash
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-ip6tables = 1
+net.bridge.bridge-nf-call-iptables = 1
+EOF
+sudo sysctl --system
+```
+
+Pastikan modul `br_netfilter` sudah dimuat sebelum melakukan langkah ini. Hal ini dilakukan dengan menjalankan `lsmod | grep br_netfilter`. Untuk memuatnya secara eksplisit gunakan `sudo modprobe br_netfilter`.
+
+Untuk detail lebih lanjut, silakan lihat laman [Network Plugin Requirements](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements).
+
+## Memeriksa port yang dibutuhkan
```

Comment:
port can translate to porta
```suggestion
## Memeriksa porta yang dibutuhkan
```

Reply:
Fixed in 0b74a869bea33a4eb86e43a89b5284bcd244444f